notion-core/src/test/scala/com/ambiata/notion/core/LocationIOSpec.scala

```diff
@@ -120,6 +121,15 @@ class LocationIOSpec extends Specification with ScalaCheck { def is = s2"""
     e <- i.exists(p)
   } yield e ==== true)
 
+  def copyFile = prop((source: LocationTemporary, target: LocationTemporary, lines: List10[String], overwrite: Boolean) => for {
+    s  <- source.location
+    t  <- target.location
+    i  <- source.io
+    _  <- i.writeUtf8Lines(s, lines.value)
```

Comment:
See new notes on other PR, we need to have these tests work on bytes not on strings. Leaving this as a note to revisit when you rebase on other change.